source/xtJucePlugin/parameterDescriptions_xt.json

```diff
@@ -716,43 +729,43 @@
 		],
 		"glideType":
 		[
-			"Portamento", "Glissando", "Fingered Porta", "Glissando", "Fingered Gliss."
+			"Portamento", "Glissando", "Fingered Porta", "Fingered Gliss"
 		],
 		"glideMode":
 		[
-			"Exp", "Linear"
+			"Exponential", "Linear"
 		],
 		"arpMode":
 		[
-			"Off", "On", "Hold", "Sound"
+			"Off", "On", "Hold"
```

Comment:
note that this value list "arpMode" is used for the multi arp mode parameter too, there it has a range of 0-3 so the value "Sound" needs to stay.

Reply:
```
loadJson@423: ParameterDescription parsing issues:
MI0ArpActive: value list arpMode contains only 3 entries but parameter range is 0-3
MI1ArpActive: value list arpMode contains only 3 entries but parameter range is 0-3
MI2ArpActive: value list arpMode contains only 3 entries but parameter range is 0-3
MI3ArpActive: value list arpMode contains only 3 entries but parameter range is 0-3
MI4ArpActive: value list arpMode contains only 3 entries but parameter range is 0-3
MI5ArpActive: value list arpMode contains only 3 entries but parameter range is 0-3
MI6ArpActive: value list arpMode contains only 3 entries but parameter range is 0-3
MI7ArpActive: value list arpMode contains only 3 entries but parameter range is 0-3
```

Reply:
Oh let me fix that in a subsequent PR!

Reply:
![image](https://github.com/dsp56300/gearmulator/assets/87139854/edee88a7-3e1e-4d22-aac7-d9e82d36445c)

Added a message box so that you can see the errors in non-debug builds too

Reply:
Oh you reinstated "Sound", thanks!